etna/models/nn/tft_native/tft.py

```diff
@@ -683,9 +645,9 @@ def params_to_tune(self) -> Dict[str, BaseDistribution]:
         """
         return {
             "num_layers": IntDistribution(low=1, high=3),
-            "n_heads": IntDistribution(low=1, high=3),
-            "hidden_size": IntDistribution(low=4, high=64, step=4),
-            "lr": FloatDistribution(low=1e-5, high=1e-2, log=True),
-            "encoder_length": IntDistribution(low=1, high=20),
-            "dropout": FloatDistribution(low=0.0, high=0.3),
+            "n_heads": CategoricalDistribution([1, 4]),
+            "hidden_size": CategoricalDistribution([16, 20, 40, 80, 160, 240, 320]),
+            "lr": FloatDistribution(low=1e-4, high=1e-2, log=True),
+            "dropout": FloatDistribution(low=0.1, high=0.9, step=0.1),
+            "train_batch_size": IntDistribution(low=64, high=256, log=True),
```

Comment:
First of all, it was categorical distribution in the paper, but do we really need to optimize it? In other nns we don't perform batch_size optimisation